src/systemdhook.c

```diff
@@ -273,6 +227,37 @@ static char *get_process_cgroup_subsystem_path(int pid, const char *subsystem) {
 	return NULL;
 }
 
+/* reads filesystems from @tb (libmount) looking for cgroup file systems
```

Comment:
This comment looks incorrect. We are bind mounting cgroup root and ensuring that systemd_path is created before making the mount read-only.